### PR TITLE
Update chart-features-markers.md

### PR DESCRIPTION
**Pull request by mddifilippo89**
mdd-seo-chart-markers

used existing marker-options sample and marker content from 
https://staging.infragistics.com/blazorsite/components/category-chart-config-options.html#configuring-chart-markers

### CODE REVIEW

File: doc/en/components/charts/chart-features-markers.md [hunk 1]
```diff
@@ -7,7 +7,57 @@ mentionedTypes: []
...
+
+```html
+<igx-category-chart
+    [dataSource]="data"
+    width="700px"
+    height="500px"
+    markerType="diamondMarker"
+    markerBrushes="red"
+    markerOutlines="yellow">
+</igx-category-chart>
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/chart-features-markers.md:43]: "diamondMarker" is invalid value for markerType. Use "Diamond"

File: doc/en/components/charts/chart-features-markers.md [hunk 2]
```diff
@@ -7,7 +7,57 @@ mentionedTypes: []
...
+```
+
+```tsx
+ <IgrCategoryChart dataSource={this.state.data}
+    width="700px"
+    height="500px"
+    markerType="diamondMarker"
+    markerBrushes="red"
+    markerOutlines="yellow" />
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/chart-features-markers.md:52]: "diamondMarker" is invalid value for markerType. Use "Diamond"

File: doc/en/components/charts/chart-features-markers.md [hunk 3]
```diff
@@ -7,7 +7,57 @@ mentionedTypes: []
...
+```
+```html
+<igc-category-chart
+    id="chart"
+    width="700px"
+    height="700px"
+    marker-type="diamondMarker"
+    marker-brushes="red"
+    marker-outlines="yellow">
+</igc-category-chart>
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/chart-features-markers.md:62]: "diamondMarker" is invalid value for markerType. Use "Diamond"

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: doc/en/components/charts/chart-features-markers.md (hunk 3 above)
Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-markers.md:62]: pending

File: doc/en/components/charts/chart-features-markers.md (hunk 2 above)
Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-markers.md:52]: pending

File: doc/en/components/charts/chart-features-markers.md (hunk 1 above)
Comment (mddifilippo89) [reply doc/en/components/charts/chart-features-markers.md:43]: pending

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**